clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/ResizeAsgAtomicOperationConverter.groovy

```diff
@@ -38,4 +40,36 @@ class ResizeAsgAtomicOperationConverter extends AbstractAtomicOperationsCredenti
     converted.credentials = getCredentialsObject(input.credentials as String)
     converted
   }
+
+  /**
+   * Converts legacy resize asg descriptions to their modern equivalent.
+   *
+   * Expects to be run after {@see AsgNameToServerGroupNameDescriptionPreProcessor}.
+   */
+  @Order(0)
```

Comment:
Wonder if its worth doing something similar to this instead of using the `@Order`:

http://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/autoconfigure/AutoConfigureAfter.html